### DISCUSSION TITLE
Question about adding a DS18B20 Temp sensor to iFan03

### DISCUSSION THREAD

**Discussion by pvanderwal** [arendst/Tasmota#10480] [Q&A] [answered]
I'm using an iFan03 in my living room, but I'm using a separate dimmer module for the light.  

Since I'm not using the light control built into the iFan03, I removed the relay and the diode.  I'd like to connect a DS18B20 to this GPIO.
Is this possible with the stock Tasmota firmware or will I have to build a custom firmware?  

If I need to build a custom firmware, is there instructions somewhere on how to do that with Tasmota?   I have some experiece working with Arduino's and ESP-8266 using the Arduino IDE.

**Comment by sfromis** [comment, marked as answer]
https://github.com/arendst/Tasmota/blob/development/BUILDS.md shows that USE_DS18x20 is included in most standard firmware, except for lite and ir.

If you had to compile (should not be needed) the Arduino IDE is no more supported https://tasmota.github.io/docs/Compile-your-build/
